recipes-core/rauc/rauc/mx8mm/system_emmc.conf

```diff
@@ -1,47 +1,48 @@
 [system]
 compatible=@MACHINE@
-bootloader=uboot
+bootloader=@BOOTLOADER@
 mountprefix=/mnt/rauc
 
 [handlers]
-pre-install=/usr/bin/rauc_downgrade_barrier.sh
+pre-install=/usr/lib/rauc/rauc-pre-install.sh
+post-install=/usr/lib/rauc/rauc-post-install.sh
 
 [keyring]
-path=ca.cert.pem
+path=@RAUC_KEYRING_FILE@
 
 # Bootloader
 [slot.bootloader.0]
-device=/dev/mmcblk2
+device=/dev/mmcblk@EMMC_DEV@
 type=boot-emmc
 
 # System A
 [slot.rootfs.0]
-device=/dev/mmcblk2p5
+device=/dev/mmcblk@EMMC_DEV@p5
 type=ext4
 bootname=system0
 
 [slot.boot.0]
-device=/dev/mmcblk2p1
+device=/dev/mmcblk@EMMC_DEV@p1
 type=vfat
 parent=rootfs.0
 
 [slot.appfs.0]
-device=/dev/mmcblk2p7
+device=/dev/mmcblk@EMMC_DEV@p7
```

Comment:
maybe the appfs could be encrypted too, but it is not a feature for the default BSP.

Reply:
Yes, for the moment I would leave it as is for the appfs. Implementing file system encryption for the app partition also requires some additional changes in our provisioning and encryption tools I guess.